dbt_coves/templates/source_model_props.yml

```diff
@@ -14,10 +14,9 @@ models:
   - name: {{ model.lower() }}
     columns:
 {%- for cols in nested.values() %}
-  {%- for col in cols %}
-      - name: {{ col.lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
-  {%- endfor %}
+      - name: {{ cols["key"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
+      - description: {{ cols["description"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
```

Comment:
The dash ("-") indicates a new item. In this case, description is not a new item. Example:

models:
  - name: _airbyte_raw_country_populations
    columns:
      - name: country_code
         description: country_code_
      - name: country_name
         description: country_name_
      - name: value
         description: value_
@LeonardoPelanda